packs/data/feats.db/skill-mastery-rogue.json

```diff
@@ -33,7 +33,340 @@
                 }
             ]
         },
-        "rules": [],
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.SkillAcr",
+                        "predicate": {
+                            "all": [
+                                "skill:acr:rank:2"
+                            ]
+                        },
+                        "value": "acr"
```

Comment:
If we are moving towards full name, as stated above, should all these change?

Reply:
They'll get migrated at some point, but we can always just do a larger PR to catch all of them.